scripts/push_benchmark.sh

```diff
@@ -0,0 +1,54 @@
+#!/usr/bin/env bash
+
+export YLW='\033[1;33m'
+export RED='\033[0;31m'
+export GRN='\033[0;32m'
+export BLU='\033[0;34m'
+export BLD='\033[1m'
+export RST='\033[0m'
+
+# Clear line
+export CLR='\033[2K'
+
+set -o nounset
+set -o errexit
+set -o pipefail
+
+REPO_URL="git@github.com:status-im/status-app-benchmarks.git"
+
+GIT_ROOT=$(cd "${BASH_SOURCE%/*}" && git rev-parse --show-toplevel)
+
+echo -e "${GRN}Pushing benchmark results${RST}"
+
+cd "${GIT_ROOT}"
+# Get the commit SHA from the status-app repo BEFORE cloning benchmarks-repo
+commit_sha=$(git rev-parse --short HEAD)
+
+git clone "${REPO_URL}" benchmarks-repo
+cd benchmarks-repo
+
+date_time=$(date -u '+%Y-%m-%dT%H:%M:%S')
+
+echo -e "${GRN}Creating virtual environment${RST}"
+python3 -m venv .venv
```

Comment:
do we need to start vitual env here when you already enter virtual env in Deps stage in Jenkinsfile?

Reply:
Okay this virtual env is for different location i see